docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,266 @@
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Oracle Cloud](../../../img/database-access/guides/oracle_selfhosted_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- Self-hosted Oracle server instance 18c or later.
+- The `sqlcl` [Oracle client](https://www.oracle.com/pl/database/sqldeveloper/technologies/sqlcl/) installed and added to your system's `PATH` environment variable or any GUI client that supports JDBC
+  Oracle thin client.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+
+## Step 1/5. Start the Database Service
+
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+<Tabs>
+<TabItem label="With CLI flags">
+
+You can start the Teleport Database Service without configuration file using a
+CLI command:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db start \
+   --token=/tmp/token \
+   --auth-server=teleport.example.com:3080 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+Note that the `--auth-server` flag must point to the Teleport cluster's Proxy
+Service endpoint because the Database Service always connects back to the
+cluster over a reverse tunnel.
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db start \
+   --token=/tmp/token \
+   --auth-server=mytenant.teleport.sh:443 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+Note that the `--auth-server` flag must point to your Teleport Cloud tenant
+address.
+
+</ScopedBlock>
+
+</TabItem>
+<TabItem label="Using a config file">
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=teleport.example.com:3080 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=mytenant.teleport.sh:443 \
+   --name=test \
+   --protocol=oracle \
+   --uri=oracle.example.com:2484 \
+   --labels=env=dev
+```
+
+</ScopedBlock>
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple different services, for example
+  multiple Database Service agents as well as the SSH Service or Application
+  Service.
+
+</Admonition>
+
+Start the database service:
+
+```code
+$ teleport start --config=/path/to/teleport-db.yaml --token=/tmp/token
+```
+
+</TabItem>
+</Tabs>
+
+<Admonition type="tip">
```

Comment:
To match our style guide, let's put the text of this tip in the Next Steps section instead. At the same time, it might make sense to recommend using a configuration file for all users so it's easier to set up a systemd service.